xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/src/main/resources/Mail/MailConfig.xml

```diff
@@ -43,12 +43,168 @@
+        <name>host</name>
+        <number>3</number>
+        <picker>0</picker>
+        <prettyName>Host</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </host>
+      <password>
+        <algorithm/>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <name>password</name>
+        <number>6</number>
+        <picker>0</picker>
+        <prettyName>Password</prettyName>
+        <size>30</size>
+        <storageType>Clear</storageType>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.PasswordClass</classType>
+      </password>
+      <port>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <name>port</name>
+        <number>4</number>
+        <numberType>integer</numberType>
+        <prettyName>Port</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+      </port>
+      <properties>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>properties</name>
+        <number>7</number>
+        <picker>0</picker>
+        <prettyName>JavaMail Properties</prettyName>
+        <rows>5</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </properties>
+      <sendWaitTime>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <name>sendWaitTime</name>
+        <number>8</number>
+        <numberType>long</numberType>
+        <prettyName>Send Wait Time</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+      </sendWaitTime>
+      <username>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <name>username</name>
+        <number>5</number>
+        <picker>0</picker>
+        <prettyName>Username</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </username>
+    </class>
+    <property>
+      <from/>
+    </property>
+    <property>
+      <host>127.0.0.1</host>
+    </property>
+    <property>
+      <password/>
+    </property>
+    <property>
+      <port>25</port>
+    </property>
+    <property>
+      <properties/>
+    </property>
+    <property>
+      <username/>
+    </property>
+  </object>
```

Comment:
Something went wrong in this commit! As you can see, there is extra content added while it shouldn't. Have you used the latest source code?